references/classification/presets.py

```diff
@@ -0,0 +1,37 @@
+from torchvision.transforms import autoaugment, transforms
+
+
+class ClassificationPresetTrain:
+    def __init__(self, crop_size, mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225), hflip_prob=0.5,
+                 auto_augment_policy=None, random_erase_prob=0.0):
+        trans = [transforms.RandomResizedCrop(crop_size)]
+        if hflip_prob > 0:
+            trans.append(transforms.RandomHorizontalFlip(hflip_prob))
+        if auto_augment_policy is not None:
+            aa_policy = autoaugment.AutoAugmentPolicy(auto_augment_policy)
+            trans.append(autoaugment.AutoAugment(policy=aa_policy))
+        trans.extend([
+            transforms.ToTensor(),
+            transforms.Normalize(mean=mean, std=std),
+        ])
+        if random_erase_prob > 0:
+            trans.append(transforms.RandomErasing(p=random_erase_prob))
+
+        self.transforms = transforms.Compose(trans)
+
+    def __call__(self, img):
+        return self.transforms(img)
+
+
+class ClassificationPresetEval:
+    def __init__(self, crop_size, resize_size=256, mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)):
+
+        self.transforms = transforms.Compose([
```

Comment:
Usually, for text domain, we will need to download the transform, for example sentencepiece model, or a vocabulary saved in text file.

Reply:
I'm writing here what we discussed on the call.

It seems that supporting your case is possible by using PyTorch Hub's `load_state_dict_from_url()` method and then passing the result to your code. This is very common pattern in TorchVision, used mainly for pre-trained models. Example:
https://github.com/pytorch/vision/blob/97885cb1c983525dd14b82f38760221cd4ea7a17/torchvision/models/mobilenetv3.py#L244-L249